gt_concordance/gt_concordance.py

```diff
@@ -0,0 +1,519 @@
+    if not os.path.exists(config):
+        click.echo('Config file does not exist!')
+        exit(1)
+    with open(config, 'r') as config_file:
+        config = yaml.load(config_file) or {}
+    context.obj = config
+
+@cli.command()
+@click.pass_context
+def parse_xl_files(context):
+    config = context.obj
+    # checking config
+    XL_FILES_PATH = config.get('XL_FILES_PATH')
+    if XL_FILES_PATH is None:
+        click.echo("config file missing XL_FILES_PATH argument")
+        click.echo('Do you want to enter path to excel files? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter path to excel files')
+            XL_FILES_PATH = raw_input()
+        else:
+            click.echo('No XL_FILES_PATH given. Terminating')
+            exit(0)
+    if not os.path.exists(XL_FILES_PATH):
+        click.echo("Path to excel files does not exist! Path: {}".format(XL_FILES_PATH))
+        click.echo('Terminating')
+        exit(0)
+    #
+    ANALYSIS_PATH = config.get('ANALYSIS_PATH')
+    if ANALYSIS_PATH is None:
+        click.echo("config file missing ANALYSIS_PATH")
+        click.echo('Do you want to enter analysis path? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter the output path')
+            ANALYSIS_PATH = raw_input()
+        else:
+            exit(0)
+    if not os.path.exists(ANALYSIS_PATH):
+        click.echo('Analysis path does not exist! Path: {}'.format(ANALYSIS_PATH))
+        click.echo('Terminating')
+        exit(0)
+
+    XL_FILES_ARCHIVED = config.get('XL_FILES_ARCHIVED')
+    if XL_FILES_ARCHIVED is None:
+        click.echo('config file missing XL_FILES_ARCHIVED')
+        click.echo('Do you want to enter path where to archive excel files? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            XL_FILES_ARCHIVED = raw_input()
+        else:
+            click.echo('Please enter where to archive excel files')
+            XL_FILES_ARCHIVED = raw_input()
+    if not os.path.exists(XL_FILES_ARCHIVED):
+        click.echo('Path does not exist! Path: {}'.format(XL_FILES_ARCHIVED))
+        click.echo('Terminating')
+        exit(0)
+
+    click.echo('Looking for .xlsx files in {}'.format(XL_FILES_PATH))
+    xl_files = [os.path.join(XL_FILES_PATH, filename) for filename in os.listdir(XL_FILES_PATH) if '.xlsx' in filename]
+    if not xl_files:
+        click.echo('No .xlsx files found! Terminating')
+        exit(0)
+
+    click.echo('Following files are found: {}'.format(', '.join([os.path.basename(xl_file) for xl_file in xl_files])))
+
+    # parsing snps file
+    snps_data = parse_maf_snps_file(config)
+
+    # parsing xl files
+    genotype_data = {}
+    for xl_file in xl_files:
+        click.echo("Parsing file: {}".format(os.path.basename(xl_file)))
+        data = pyexcel_xlsx.get_data(xl_file)
+        data = data.get('HaploView_ped_0') # sheet name
```

Comment:
it's not the first sheet. it's 5th or maybe 9th, something like that. And it *seems* to be always the same. Maybe at a certain point they decide to change the format, well, yes then this code won't work anymore. but it's hard to write it in more general way, it's so many sheets...